amazon-kinesis-client/src/main/java/software/amazon/kinesis/leases/LeaseManagementConfig.java

```diff
@@ -104,13 +104,14 @@ public class LeaseManagementConfig {
     private long failoverTimeMillis = 10000L;
 
     /**
-     * Multiplier for the failoverTimeMillis in which leases which are expired for an extended period of time defined by
-     * (agedFailoverTimeMultiplier * failoverTimeMillis) are taken with priority, disregarding the target
-     * but obeying the maximum limit per worker.
+     * Whether workers should take very expired leases at priority. A very expired lease is when a worker does not
+     * renew its lease in 3 * {@link LeaseManagementConfig#failoverTimeMillis}. Very expired leases will be taken at
```

Comment:
Can you also mentioned newly created leases due to shard mutation that are not assigned to any worker are also considered very-expired ?

Reply:
Updated